packages/commerce-sdk-react/src/hooks/ShopperBaskets/action.ts

```diff
@@ -334,9 +334,11 @@ the body are the following properties if specified:
 /**
  * A hook for performing actions with the Shopper Baskets API.
  */
-export function useShopperBasketsAction<Action extends ShopperBasketsActions>(
+// TODO: Why does prettier not like "extends `${Actions}`"?
+// eslint-disable-next-line prettier/prettier
```

Comment:
The fix for this `TODO` is a major version bump to prettier (we're 2 years behind!). Since that involves formatting changes to a bunch of unrelated files, I'll leave it to a separate PR.